### DISCUSSION TITLE
文档网站无法访问

### DISCUSSION THREAD

**Discussion by FuDujilm** [bs-community/blessing-skin-server#612] [Q&A]
ip地域：四川

仓库中提到的文档网页无法访问https://blessing.netlify.app/
itdog全国ping测试
![image](https://github.com/bs-community/blessing-skin-server/assets/72288165/e6001b8a-7b0a-4234-9829-929c3a927c13)
（很明显是被ICMP禁ping了）
网站测速图
![image](https://github.com/bs-community/blessing-skin-server/assets/72288165/3eea25ec-f2a2-4bdd-bee3-d72f37104836)
![image](https://github.com/bs-community/blessing-skin-server/assets/72288165/2af68223-510e-4092-900f-a00577ee875b)
根据上图可知，四川地方被解析到了127.0.0.1，这是什么操作（

**Comment by tnqzh123** [comment]
懂的都懂的操作，解决方案懂的都懂，不懂的我也不好多说

**Reply by FuDujilm** [reply]
> 懂的都懂的操作，解决方案懂的都懂，不懂的我也不好多说

这边使用vpn无法也没办法，解决办法是用cf搭了个反代（
